packages/amplify-provider-awscloudformation/src/utility-functions.js

```diff
@@ -423,4 +424,8 @@ module.exports = {
 
     return results;
   },
+  /**
+   * Provides the same AWS config used to push the amplify project
+   */
+  retrieveAwsConfig: async (context) => getAwsConfig(context),
```

Comment:
What's the purpose of this pass through function? Why not call `getAwsConfig()` directly?

Reply:
@marcvberg I know that you introduced this, can you answer this one?

Reply:
The reasoning is to continue exposing it through execute provider utils rather than explicitly creating a dependency on the package since that's the pattern we've been following from the API category